homeassistant/components/fibaro/sensor.py

```diff
@@ -80,112 +106,73 @@ async def async_setup_entry(
 ) -> None:
     """Set up the Fibaro controller devices."""
     entities: list[SensorEntity] = []
+
     for device in hass.data[DOMAIN][entry.entry_id][FIBARO_DEVICES][Platform.SENSOR]:
-        entities.append(FibaroSensor(device))
+        entity_description = None
+        for desc in MAIN_SENSOR_TYPES:
+            if desc.key == device.type:
```

Comment:
Instead of this loop, can we make `MAIN_SENSOR_TYPES` a dict?

Reply:
Yes a dict is better, I just haven't found an example and I thought that would be discouraged. I changed it.